### PR TITLE
Add argument validation to 'ServerResponse.prototype.writeHead'

### PR DESCRIPTION
**Pull request by LaszloLango**
IoT.js-DCO-1.0-Signed-off-by: László Langó llango.u-szeged@partner.samsung.com

### CODE REVIEW

File: test/node/parallel/test-http-write-head.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+function runTest() {
+  http.get({ port: this.address().port }, common.mustCall(function (response) {
+    response.on('end', common.mustCall(function () {
+      assert.strictEqual(response.headers['test'], '1' /*'2'*/);
+      // assert.notStrictEqual(response.rawHeaders.indexOf('Test'), -1);
+      s.close();
+    }));
+    response.resume();
+  }));
+}
```
Comment (zherczeg) [inline test/node/parallel/test-http-write-head.js]: Newline needed.

Comment (LaszloLango) [inline test/node/parallel/test-http-write-head.js]: fixed

**Comment by yichoi** [comment]
@LaszloLango   please rebase it to the master

**Comment by yichoi** [comment]
LGTM

**Pull request merged**